src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -76,6 +76,13 @@ def register_options(cls, register):
             advanced=True,
             help="The format of the generated XML file. See https://docs.pytest.org/en/latest/reference.html#confval-junit_family.",
         )
+        register(
+            "--execution-slot-var",
+            type=str,
+            default="",
+            advanced=True,
+            help="If a non-empty string, the process execution slot id (an integer) will be exposed to tests under this environment variable name.",
```

Comment:
Nit: I think this might be longer than 100 characters. Appreciated if you can split this up into multiple lines. I like the style of wrapping in parentheses:

```python
help=(
    "blah blah"
    "blah"
)
```

Reply:
Wouldn't this be caught by the linter if it was?

Reply:
Nope. We turned off the check for Flake8 because it doesn't work well with Black. Black is careful when formatting strings and doesn't touch them generally.